source/validations.ts

```diff
@@ -172,6 +172,25 @@ export class Validations {
 		})
 	}
 
+	/**
+	 * Warns the user that the behaviour for `max: 0` is changing in the next
+	 * major release.
+	 *
+	 * @param max {number} - The maximum number of hits per client.
+	 *
+	 * @returns {void}
+	 */
+	// TODO: Remove in v7.0.0
```

Comment:
Rather than removing it, I think we'll just update the language in v7.0.0 to clarify that the behavior has now changed. Maybe remove it in a future 7.x release, but keep it there for now in case someone jumps from 6.9.0 to 7.0.0

Reply:
That makes sense, alright!